S23/b-sai/main.py

```diff
@@ -0,0 +1,50 @@
+
+def to_lowercase(word: str, language: str):
+    """
+    word: str, the string to be converted to lowercase
+    language: str, the language of the string, in BCP 47 format
+    """
+    result = ""
+    
+    if language.startswith(("zh", "th", "ja")):
+        return word.lower()
```

Comment:
And the point was in these cases to not bother calling lower() as an optimization.